src/ripple/app/tx/impl/SetSignerList.cpp

```diff
@@ -182,7 +182,7 @@ removeSignersFromLedger(
     ApplyView& view,
     AccountRootKeylet const& accountKeylet,
     Keylet const& ownerDirKeylet,
-    Keylet const& signerListKeylet,
+    KeyletBase const& signerListKeylet,
```

Comment:
I think I can specialize this parameter to `SignersKeylet`. Should I do that or leave it as is? Since this function is specific to signers, I don't think there is any harm in narrowing it's scope.